src/FsAutoComplete.Logging/FsAutoComplete.Logging.fsproj

```diff
@@ -1,10 +1,14 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
Needed to get the FsOpenTelemetry diagnostics to play nice.

Reply:
this is fine - we're not targeting net472 anymore so targeting the full TFM makes sense.